src/translation.cpp

```diff
@@ -416,3 +416,8 @@ std::string operator+( const translation &lhs, const translation &rhs )
 {
     return lhs.translated() + rhs.translated();
 }
+
+bool operator<( const translation &lhs, const translation &rhs )
```

Comment:
Please use `translation::translated_lt` for localized comparison. Defining `<` as the localized one will be inconsistent with `==` and `!=` defined in `class translation`.

Reply:
Thanks for taking the time to review. That makes sense. 

I'm pretty sure localized_comparator will work, if scoping ends up being an issue I'll figure something out.

I'm on vacation for the next week and can't compile so I'll probably not get to this but just wanted to let you know I'm not ignoring the suggestion 😄